src/cdk/table/table.spec.ts

```diff
@@ -278,6 +280,18 @@ describe('CdkTable', () => {
     expect(getRows(tableElement).length).toBe(0);
   }));
 
+  it('should be able to render multiple header and footer rows', () => {
+    setupTableTestApp(MultipleHeaderFooterRowsCdkTableApp);
+    fixture.detectChanges();
+
+    expectTableToMatchContent(tableElement, [
+      ['first-header'],
+      ['second-header'],
+      ['first-footer'],
+      ['second-footer'],
+    ]);
+  });
+
```

Comment:
Should there be a test for when the header/footer rows change?

Reply:
Yup - added a new test for that case